### PR TITLE
Create 14-translations.md

### PR DESCRIPTION
**Pull request by rtymchyk**

### CODE REVIEW

File: 14-translations.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#### Singular
+```javascript
+<LocalizedMessage message={_('Hello')} />
+```
+
+#### Plural
+```javascript
+const friendCount = ...;
+
+<LocalizedMessage
```
Comment (tildedave) [inline 14-translations.md]: For each of these, probably best to have some examples about what will be output in different situations.  e.g. 1 user, 2 users, etc

File: 14-translations.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Translations
```
Comment (tildedave) [inline 14-translations.md]: So something I think is pretty fundamental to translations is the notion that (in my phrasing) you can't translate partial sentences.  That's where a lot of the need for HTML in translated message strings comes from ... and why I had to do some awful things for the start page.  Might be a good thing to call out what the "minimum translatable unit" is with some examples from our codebase.

**Comment by rtymchyk** [comment]
@tildedave Added more info on best practices as well as examples.

File: 14-translations.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+|  | 1 | Facebook (1) friend |
+|  | 2 | Facebook (2) friends |
+| fr-FR | 0 | Facebook (0) ami |
+|  | 1 | Facebook (1) ami |
+|  | 2 | Facebook (2) amis |
+
+Notice that for the French language, the plural rules is different from English. In some languages there is no distinction between singular and plural, and for others there are 6 plural forms with special rules.
+
+## Best Practices
+1. <b>Extract only complete strings</b>
```
Comment (tildedave) [inline 14-translations.md]: Use Markdown `*` to bold in markdown

Comment (tildedave) [inline 14-translations.md]: *test*

Comment (tildedave) [inline 14-translations.md]: **test**

Comment (tildedave) [inline 14-translations.md]: `**test**`

**Comment by tildedave** [comment]
this looks awesome!

**Pull request merged**